crnk-core/src/main/java/io/crnk/core/repository/foward/strategy/GetFromOwnerStrategy.java

```diff
@@ -49,7 +49,7 @@
 
 			QuerySpec idQuerySpec = new QuerySpec(targetInformation);
```

Comment:
I guess idQuerySpec can also be removed then?

Reply:
Yes, I've forgot remove it